backend-ruby/lib/photobank/entities/postdata.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class PostData < Hanami::Entity
+end
```

Comment:
Ruby convention is for the file names to match the class, so `PostData` would be `post_data.rb`, missing an underscore